### ISSUE TITLE
Font styles not working for sidebar text in IE11

### ISSUE THREAD

**Issue by kaiyoma** [storybooks/storybook#757]
The `font-family` style is not being applied correctly to the text in the sidebar in IE11, causing the text to be rendered in a simple Times font. This is true for both the dev server and static versions of Storybook. To repro, open this URL in IE11 and look at the text in the sidebar: http://airbnb.io/react-dates/

**Comment by ndelangen** [comment]
Good find! This is the CSS rule being used:
```css
    font-family: -apple-system, ".SFNSText-Regular", "San Francisco", Roboto, "Segoe UI", "Helvetica Neue", "Lucida Grande", sans-serif;
```

Do you have a suggestion what would improve it for IE11 / windows users?

**Comment by kaiyoma** [comment]
Hmmm, after digging into this a bit, I'm actually not sure what the problem is. For starters, it's an issue with IE11 only; Chrome, Edge, and Firefox all render the font correctly in Windows. There are no errors in the console.

I initially thought IE11 was choking on `-apple-system`, but if I create a static HTML page with the same markup and the same styles, IE actually renders everything correctly!

The only "weird" thing I see about the style string (that could possibly trip up IE) is that there's a space before the trailing semi-colon, but that's really stretching. For a second, I thought maybe IE required `-apple-system` to be in quotes, but I'm not entirely convinced of that.

**Comment by Hypnosphi** [comment]
Looks like it's fixed with #1560: my IE11 renders Segoe UI in sidebar
